src/main/frontend/components/repo.cljs

```diff
@@ -90,93 +71,6 @@
   (page-handler/create-today-journal!)
   (shortcut/refresh!))
 
-(rum/defc sync-status < rum/reactive
-  {:did-mount (fn [state]
-                (js/setTimeout common-handler/check-changed-files-status 1000)
-                state)}
-  [repo]
-  (when (and repo
-             (string/starts-with? repo "https://"))
-    (let [changed-files (state/sub [:repo/changed-files repo])
-          should-push? (seq changed-files)
-          git-status (state/sub [:git/status repo])
```

Comment:
I see `:git/status` and a couple other :git keys in db-schema. Should those be removed?